clib/SeisintDatasetSAXHandler.h

```diff
@@ -278,6 +280,8 @@ class CLIB_API SeisintDatasetHandlerImpl : public StackHandlerImpl
                     CComPtr<ITable> table = m_tableMap[m_datasetName].m_table.get();
                     table->SetHasChildDataset(true);
                     std::_tstring content, _content = e.GetContent();
+                    boost::algorithm::replace_all(_content, _T("\t"), _T("    "));
+                    content = _content;
```

Comment:
I don't think this does anything at all.  the escapexml call is going to overwrite it anyway.  Much cleaner would be something like:

```
                    std::_tstring rawContent = e.GetContent();
                    boost::algorithm::replace_all(rawContent, _T("\t"), _T("    "));
                    std::_tstring escapedContent;
                    table->AppendCell(m_row, m_col, EscapeXML(rawContent, escapedContent));
```

Reply:
@ghalliday yes. Also didn't think of that.

Reply:
That particular comment was about the content = _content.  It would have worked, it was just unnecessary/redundant and confusing.

Reply:
I always try to make the least amount of changes to the code when modifying. I think in this case, I should have rewritten it like you did. Shouldn't always follow that rule.